cmd/influx_stress/influx_stress.go

```diff
@@ -7,9 +7,11 @@ import (
 	"runtime/pprof"
 
 	"github.com/influxdata/influxdb/stress"
+	v2 "github.com/influxdata/influxdb/stress/v2"
```

Comment:
both of the packages are named `stress` so I aliased. Should I rename the new package to `v2`?